net_http_mux.go

```diff
@@ -23,23 +23,23 @@ import (
 //	})
 //	s.Run()
 func Group(s *Server, path string, routeOptions ...func(*BaseRoute)) *Server {
-	if path == "/" {
+	switch {
+	case path == "/":
 		path = ""
-	} else if path != "" && path[len(path)-1] == '/' {
-		slog.Warn("Group path should not end with a slash.", "path", path+"/", "new", path)
+	case strings.HasSuffix(path, "/"):
+		slog.Warn("Group path should not end with a slash.", "path", path, "new", strings.TrimSuffix(path, "/"))
 	}
```

Comment:
Maybe you could use CutSuffix that returns a bool flag

This way it would look like this (pseudo code)

```suggestion
	case cleaned, found := strings.CutSuffix(path, "/"); found:
		slog.Warn("Group path should not end with a slash.", "path", path, "new", cleaned)
	}
```

Reply:
How does that look now? You can't instantiate inside case statement AFAIK.